dist/leaflet.geometryutil.js

```diff
@@ -498,6 +498,78 @@ L.GeometryUtil = L.extend(L.GeometryUtil || {}, {
     },
 
     /**
+     * Concatenate polylines whose start/end points are within a close distance
+     * @param  {L.Map}        map
+     * @param  {L.Polyline[]} polylines         An array of L.Polyline objects
+     * @param  {Number}       [tolerance=15]    A pixel distance in which points will be considered equal
+     * @return {L.Polyline[]} concateLines      An array of concatenated L.Polyline objects
+     */
+    concatLines: function(map, polylines, tolerance) {
```

Comment:
It reminds me this :) https://github.com/makinacorpus/Geotrek/blob/master/geotrek/core/tests/test_path_merge.py#L12 /cc @gutard @fredericbonifas